### PR TITLE
Add additional message when backendfs is extfs without d_type support

### PR DESCRIPTION
**Pull request by cgxu519**
Signed-off-by: Chengguang Xu <cgxu519@gmx.com>

Attempt to provide more meaningful warning message.

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @kolyshkin PTAL

**Comment by cgxu519** [comment]
ping @kolyshkin

**Review by thaJeztah — APPROVED** [review]
LGTM

File: daemon/graphdriver/overlayutils/overlayutils.go [hunk 1]
```diff
@@ -14,6 +14,11 @@ func ErrDTypeNotSupported(driver, backingFs string) error {
 	if backingFs == "xfs" {
 		msg += " Reformat the filesystem with ftype=1 to enable d_type support."
 	}
+
+	if backingFs == "extfs" {
+		msg += " Reformat the filesystem with -O filetype to enable d_type support."
```
[the remark this answers is not in the document]
Comment (kolyshkin) [reply daemon/graphdriver/overlayutils/overlayutils.go]: Maybe even
```golang
msg += " Reformat the filesystem (or use tune2fs) with -O filetype flag to enable d_type support."
```

[the remark this answers is not in the document]
Comment (cgxu519) [reply daemon/graphdriver/overlayutils/overlayutils.go]: Thanks for your review, I've added more information about tune2fs.

**Review by thaJeztah — APPROVED** [review]
still LGTM, thanks!

**Comment by thaJeztah** [comment]
I see @kolyshkin's comments are addressed; I'll go ahead and merge

**Comment by thaJeztah** [comment]
Thanks for you contribution @cgxu519 !

**Pull request merged**